traffic_ops/app/db/seeds.sql

```diff
@@ -22,6 +22,257 @@ insert into role (id, name, description, priv_level) values (5, 'portal', 'Porta
 insert into role (id, name, description, priv_level) values (6, 'migrations', 'database migrations user - DO NOT REMOVE', 20) ON CONFLICT DO NOTHING;
 insert into role (id, name, description, priv_level) values (7, 'federation', 'Role for Secondary CZF', 15) ON CONFLICT DO NOTHING;
 insert into role (id, name, description, priv_level) values (8, 'steering', 'Role for Steering Delivery Services', 15) ON CONFLICT DO NOTHING;
+insert into role (id, name, description, priv_level) values (9, 'root', 'Role for full capabilities - super-user ', 30) ON CONFLICT DO NOTHING;
```

Comment:
ok, you are probably right. let's not mess with the priv_level in this PR. i can do that in another PR once i verify that changing it won't impact anything negatively.